util/gh-pages/script.js

```diff
@@ -149,27 +136,25 @@ function lintAnchor(event) {
     expandLint(id);
 }
 
+let ticked;
```

Comment:
Instead of this, why not using an array or a map where `clipboard` would be the key?

Reply:
It would be more complicated without much benefit

Reply:
I can do it in a follow-up if you prefer.

Reply:
I beg to differ, it's about correctly handling everything on having some wrong edges. Either we do it right or we don't.

Reply:
Yes, `ticked` handles the case where you click multiple copy buttons. What is incorrect about the behaviour?

Reply:
You're right, my first comment only contained the technical information, not why it was needed. My bad.